src/LendingPool.sol

```diff
@@ -376,15 +372,13 @@ contract LendingPool is Owned, TrustedProtocol, DebtToken, InterestRateModule {
      * @notice Syncs interest payments to the Lending providers and the treasury.
      * @param assets The total amount of underlying assets to be paid out as interests.
      * @dev The weight of each Tranche determines the relative share yield (interest payments) that goes to its Liquidity providers
-     * @dev The Shares for each Tranche are rounded up, if the treasury receives the remaining shares and will hence loose
-     * part of their yield due to rounding errors (neglectable small).
      */
     function _syncInterestsToLiquidityProviders(uint256 assets) internal {
         uint256 remainingAssets = assets;
 
         uint256 trancheShare;
         for (uint256 i; i < tranches.length;) {
-            trancheShare = assets.mulDivUp(weights[i], totalWeight);
+            trancheShare = assets.mulDivDown(weights[i], totalWeight);
```

Comment:
if there is a discrepancy over time, should we add a 'skim' function or the like?

Reply:
No treasury takes the remainder, mulDivDown just gives us the option to set treasury fee to 0, while mulDivUp would cause overflows in that scenario